indico/modules/events/sessions/operations.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core.db import db
+from indico.modules.events.sessions.models.sessions import Session
+
+
+def create_session(event, session_config=None):
+    """Create a new session with the information passed in the `session_config` argument"""
+    session = Session(event_new=event)
+    session.populate_from_dict(session_config or {})
+    return session
+
+
+def update_session(session, session_config):
+    """Update a session based on the information in the `session_config`"""
+    session.populate_from_dict(session_config)
+
+
+def _delete_session_timetable_entries(event_session):
+    for block in event_session.blocks:
+        for contribution in block.contributions:
+            db.session.delete(contribution.timetable_entry)
+        db.session.delete(block.timetable_entry)
+
+
+def delete_sessions(sessions):
```

Comment:
`delete_session`; the loop should be outside if you want to delete more than one